### PR TITLE
feat(vocabularies): Add support for vocabularies translation on ninjs…

### PR DESCRIPTION
**Pull request by ioanpocol**
… output

### CODE REVIEW

File: superdesk/vocabularies/vocabularies.py [hunk 1]
```diff
@@ -228,6 +228,7 @@ def get_rightsinfo(self, item):
         if not all_rights or not all_rights.get('items'):
             return {}
         try:
+            all_rights['items.'] = self.get_locale_vocabulary(all_rights.get('items'), item.get('language'))
```
Comment (petrjasek) [inline superdesk/vocabularies/vocabularies.py]: seems like a typo `items.`

File: tests/vocabularies_tests.py [hunk 2]
```diff
@@ -100,5 +100,17 @@ def test_get_rightsinfo(self):
...
+    def test_get_locale_vocabulary(self):
+        items = [
+            {'is_active': True, 'name': 'FIXME1', 'qcode': 'f', 'subject': '',
+             'translations': {'name': {'fr': 'FIXME1-fr', 'es': 'FIXME1-es'}}},
+            {'is_active': True, 'name': 'FIXME2', 'qcode': 'f', 'subject': '',
+             'translations': {'name': {'fr': 'FIXME2-fr', 'es': 'FIXME2-es'}}}
+        ]
+        result = VocabulariesService().get_locale_vocabulary(items, 'fr')
+
+        self.assertEqual(result[0]['name'], 'FIXME1-fr')
```
Comment (petrjasek) [inline tests/vocabularies_tests.py]: can you check it like `self.assertEqual(result[0], {'name': 'FIXME1-fr', 'code': 'f'})` so we make sure there are no extra fields like translations

Comment (ioanpocol) [reply tests/vocabularies_tests.py]: Why we have to drop translations since it is already on original vocabulary?

Comment (petrjasek) [reply tests/vocabularies_tests.py]: because it shouldn't make it to the output

Comment (ioanpocol) [reply tests/vocabularies_tests.py]: I think it is not the case, the code that uses vocabularies just pick the name or any other value that is needed. The vocabulary is not put in output as a whole.

File: superdesk/publish/formatters/ninjs_formatter.py [hunk 3]
```diff
@@ -311,10 +311,13 @@ def _format_qcodes(self, items):
...
 
     def _format_place(self, items):
-        locator_map = superdesk.get_resource_service('vocabularies').find_one(req=None, _id='locators')
+        vocabularies_service = superdesk.get_resource_service('vocabularies')
+        locator_map = vocabularies_service.find_one(req=None, _id='locators')
 
         def getLabel(item):
-            locators = [l for l in locator_map.get('items', []) if l['qcode'] == item.get('qcode')]
+            locale_locators = vocabularies_service.get_locale_vocabulary(
+                locator_map.get('items'), item.get('language')) or []
```
Comment (petrjasek) [inline superdesk/publish/formatters/ninjs_formatter.py]: item here is a value from `item.place` so imo it won't have language set

File: tests/vocabularies_tests.py (hunk 2 above)
Comment (petrjasek) [reply tests/vocabularies_tests.py]: you're right, it's not used directly.

File: superdesk/publish/formatters/ninjs_formatter.py (hunk 3 above)
Comment (ioanpocol) [reply superdesk/publish/formatters/ninjs_formatter.py]: Yes, true
I updated the code

**Pull request merged**